From 46980a92bbb086588c605d0f611f6cad8892903e Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <mselmeci@wisc.edu>
Date: Wed, 24 Sep 2025 15:31:00 -0500
Subject: [PATCH] [PATCH] distrepos: support "x86_64_v2" arch

but update to not mix x86_64 and x86_64_v2 RPMs in the same repo
---
 builder/kojid | 1 +
 1 file changed, 1 insertion(+)

diff --git a/builder/kojid b/builder/kojid
index 672391b..669eccb 100755
--- a/builder/kojid
+++ b/builder/kojid
@@ -6211,6 +6211,7 @@ class createDistRepoTask(BaseTaskHandler):
     archmap = {'s390x': 's390', 'ppc64': 'ppc', 'x86_64': 'i686'}
     compat = {"i386": ("athlon", "i686", "i586", "i486", "i386", "noarch"),
               "x86_64": ("amd64", "ia32e", "x86_64", "noarch"),
+              "x86_64_v2": ("amd64", "ia32e", "x86_64_v2", "noarch"),
               "ia64": ("ia64", "noarch"),
               "ppc": ("ppc", "noarch"),
               "ppc64": ("ppc64p7", "ppc64pseries", "ppc64iseries", "ppc64", "noarch"),
-- 
2.47.3

